### ISSUE TITLE
Create a dashboard?

### ISSUE THREAD

**Issue by karthik** [ropensci/software-review-meta#12]
or a shiny app?

Right now the ruby interface is a while away, but a dashboard could provide:

- An overview of open, unassigned submissions
- A birds eye view of reviews in progress
- A panel to search reviewers based on keywords
 - Said panel also shows last time a person reviewed
- A text form to update review database (add notes, duration, other metadata)

**Comment by noamross** [comment]
It think this may be more work than its worth, especially if its an interim thing. The first two are fairly well handled by github issues, and Airtable has fine search capabilities.  Pulling together a script for last review date might be good - maybe that script could run and keep a column in the airtable populated.

**Comment by karthik** [comment]
There is no ETA on the ruby interface. There is no one available to work on it, and it could easily be a year or longer before it's there. A shiny interface could be knocked together in a few days, and it would also be a concrete way for someone with Shiny expertise to contribute to ropensci.

I suppose everything already exists (everything Whedon does could be done manually). It's just harder to keep up.

**Comment by noamross** [comment]
Fair point.  And I do agree that we should see about drawing on the community broadly for tackling this and some of the other infrastructure tasks.

**Comment by mpadge** [comment]
https://dashboard.ropensci.org
